content/application-service-api.md

```diff
@@ -444,33 +446,57 @@ achieved by including the `as_token` on a `/register` request, along
 with a login type of `m.login.application_service` to set the desired
 user ID without a password.
 
-    POST /_matrix/client/v3/register
-    Authorization: Bearer YourApplicationServiceTokenHere
+```
+POST /_matrix/client/v3/register
+Authorization: Bearer YourApplicationServiceTokenHere
+
+Content:
+{
+  "type": "m.login.application_service",
+  "username": "_irc_example"
+}
+```
```

Comment:
Since you're updating this anyway:

In other places where we give example requests, we don't say `Content`, and leave it implied. See https://pr2267--matrix-spec-previews.netlify.app/client-server-api/#user-interactive-api-in-the-rest-api for some examples.

Also, using two code blocks means we can decorate the content as json.